programs/sig-aggregator/config/attestor1.toml

```diff
@@ -0,0 +1,8 @@
+[server]
```

Comment:
Do we really need all of these? Feels like this could easily just be generated on the fly or something.